.travis.yml

```diff
@@ -4,4 +4,4 @@ rvm:
 services:
   - postgresql
 before_script:
-  - psql -c 'create database travis_ci_test;' -U postgres
+  - psql -c 'create database youth_sports_test;' -U postgres
```

Comment:
youth_sports_test?

Reply:
@thedanielvogelsang Good question. 

I believe this will resolve our issue of multiple test databases being created on our local machines. By removing 'travis_ci_test" and changing the name of the created database from 'travis_ci_test' to 'youth_sports_test' in our .travis.yml file, this should alleviate the multiple test DB issues.

Reply:
🍔